src/components/ProfileDisplay.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import CommunityGrid from './CommunityGrid';
+import StudentGrid from './StudentGrid';
+
+export default function ProfileDisplay() {
```

Comment:
This is not a good name for this. Maybe `ScholarshipsMadeSimpleSection`

Reply:
Alright sounds good the function name changed to `ScholarshipsMadeSimpleSection` ill also change the file name to ScholarshipsMadeSimpleSection.jsx